acceptance/indy_acceptance/indy-tests/test_scenarios/test_scenario_02.py

```diff
@@ -0,0 +1,157 @@
+import asyncio
+import json
+import logging
+import os
+import sys
+import time
+from indy import pool
+from indy.error import IndyError
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from utils.constant import Constant, Colors
+from utils.utils import generate_random_string
+from utils.report import HTMLReport
+from utils.report import TestReport, Status
+
+
+# -----------------------------------------------------------------------------------------
+# This will run acceptance tests that will validate the add/remove roles functionality.
+# -----------------------------------------------------------------------------------------
+
+
+class MyVars:
```

Comment:
Can we find a better name for the class?

Reply:
We restructured the source code. We put those variables into the TestBaseScenario class.
All test instances will inherit this test base class.